plugins/epos/config.ini

```diff
@@ -142,20 +142,23 @@ terms_data_model = []
 #metadata standard
 metadata_standard = ['DCAT']
 
+#Policy of metadata persistance
+metadata_persistance = []
+
+#Authentication for EPOS
+metadata_authentication = []
 
 [fairsharing]
 # username and password
 username = ['']
 
 password = ['']
-#Path is the folder path ehere the netadata or fomats is stored
-#Or if the username or password is given is what you are looking in
-metadata_path = ['static/fairsharing_metadata_standards140224.json']
 
-formats_path = ['static/fairsharing_formats260224.txt']
+#Path is the folder path where the metadata or formats are stored
 
+metadata_path = ['static/fairsharing_metadata_standards140224.json']
```

Comment:
date format used in the file name is not a good approach, better YYYYMMDD (i.e. 20240214).. same in the one below